src/Components/ServiceScene/Breakdowns/Patterns/PatternFieldLabelStatsRow.tsx

```diff
@@ -0,0 +1,82 @@
+import { css, cx } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { useStyles2 } from '@grafana/ui';
+
+const getStyles = (theme: GrafanaTheme2) => ({
+  logsStatsRow: css`
+    label: logs-stats-row;
+    margin: ${parseInt(theme.spacing(2), 10) / 1.75}px 0;
+  `,
+  logsStatsRowActive: css`
+    label: logs-stats-row--active;
+    color: ${theme.colors.primary.text};
+    position: relative;
+  `,
+  logsStatsRowLabel: css`
+    label: logs-stats-row__label;
+    display: flex;
+    margin-bottom: 1px;
+  `,
+  logsStatsRowValue: css`
+    label: logs-stats-row__value;
+    flex: 1;
+    text-overflow: ellipsis;
+    overflow: hidden;
+  `,
+  logsStatsRowCount: css`
+    label: logs-stats-row__count;
+    text-align: right;
+    margin-left: ${theme.spacing(0.75)};
+  `,
+  logsStatsRowPercent: css`
+    label: logs-stats-row__percent;
+    text-align: right;
+    margin-left: ${theme.spacing(0.75)};
+    width: ${theme.spacing(4.5)};
+  `,
+  logsStatsRowBar: css`
+    label: logs-stats-row__bar;
+    height: ${theme.spacing(0.5)};
+    overflow: hidden;
+    background: ${theme.colors.text.disabled};
+  `,
+  logsStatsRowInnerBar: css`
+    label: logs-stats-row__innerbar;
+    height: ${theme.spacing(0.5)};
+    overflow: hidden;
+    background: ${theme.colors.primary.main};
+  `,
+});
+
+export interface Props {
+  active?: boolean;
+  count: number;
+  proportion: number;
+  value?: string;
+}
+
+export const PatternFieldLabelStatsRow = ({ active, count, proportion, value }: Props) => {
+  const style = useStyles2(getStyles);
+  const percent = `${Math.round(proportion * 100)}%`;
+  const barStyle = { width: percent };
+  const className = active ? cx([style.logsStatsRow, style.logsStatsRowActive]) : cx([style.logsStatsRow]);
+
+  return (
+    <div className={className}>
+      <div className={cx([style.logsStatsRowLabel])}>
+        <div className={cx([style.logsStatsRowValue])} title={value}>
+          {value}
+        </div>
+        <div className={cx([style.logsStatsRowCount])}>{count}</div>
+        <div className={cx([style.logsStatsRowPercent])}>{percent}</div>
+      </div>
+      <div className={cx([style.logsStatsRowBar])}>
+        <div className={cx([style.logsStatsRowInnerBar])} style={barStyle} />
+      </div>
+    </div>
+  );
+};
+
+PatternFieldLabelStatsRow.displayName = 'LogLabelStatsRow';
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/displayName

I'm not super familiar with how this will work since it's non-standard, I guess there isn't any user facing functionality that uses it, but I'd rather err on the side of caution and remove it? @matyax what do you think?

Reply:
This is not needed here at all. It's used when you wrap your component in, for example, `React.memo(() => {})`.
Tools like the React developer tools use this display name to show you which component has been rendered in the tree.

E.g. https://github.com/grafana/grafana/blob/main/packages/grafana-ui/src/components/DateTimePickers/DatePicker/DatePicker.tsx#L40

Reply:
Removed.